solidity/test/BasicConsumer_test.js

```diff
@@ -90,15 +91,15 @@ contract('BasicConsumer', () => {
       await oc.fulfillData(requestId, response, {from: oracleNode})
 
       let currentPrice = await cc.currentPrice.call()
-      assert.equal(web3.toUtf8(currentPrice), response)
+      assert.equal(web3.utils.toUtf8(currentPrice), response)
```

Comment:
Probably good to introduce a similar toUtf8 helper.

Reply:
a24813e7